### PR TITLE
typo fixes

### PR DESCRIPTION
**Pull request by hashirshoaeb**
Resolving Issue #88

### CODE REVIEW

**Comment by hashirshoaeb** [comment]
reference: [help.github.com](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork) 3rd point.

**Comment by hashirshoaeb** [comment]
@naveed-ahmad kindly review it.

File: README.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ The setup instructions assume that you're using [Git Bash](https://git-scm.com/)
 4. Assign the original repository to a remote called "upstream".
 
   ```sh
-  git remote add upstream https://github.com/quran/quran.com-frontend-v2
+  git remote add upstream https://github.com/quran/quran.com-frontend-v2.git
```
Comment (naveed-ahmad) [inline README.md]: if we're going with SSH,  then probably should add the correct SSH URL. That is `git@github.com:quran/quran.com-frontend-v2.git`

Comment (hashirshoaeb) [reply README.md]: I've added, SSH URL, also found a typo in 3.

**Comment by naveed-ahmad** [comment]
thank you for PR

**Comment by hashirshoaeb** [comment]
Alhamdulillah it is such an honour.

**Pull request merged**